src/os_execd/wcom.c

```diff
@@ -168,6 +168,24 @@ size_t wcom_dispatch(char *command, size_t length, char *output){
         }
     } else if (strcmp(rcv_comm, "restart") == 0) {
         return wcom_restart(output);
+    } else if (strcmp(rcv_comm, "rlock") == 0) {
```

Comment:
I think `lock_restart` will work better, it's more descriptive (just like the function).